dlt/destinations/impl/filesystem/filesystem.py

```diff
@@ -476,7 +476,9 @@ def _to_path_safe_string(self, s: str) -> str:
         """for base64 strings"""
         return base64.b64decode(s).hex() if s else None
 
-    def _list_dlt_table_files(self, table_name: str) -> Iterator[Tuple[str, List[str]]]:
+    def _list_dlt_table_files(
```

Comment:
you have changed the bevahior of this function but not updated the other places where it is used, this will no longer just list the files of the current pipeline by default, so please double check that there are no suprising side effects :)

Reply:
The function behaviour hasn't change for the other places because `pipeline_name` is set to None by default.

And the condition will always be True when pipeline_name is None - returning all the files wihtout filtering. The functions `_iter_stored_schema_files` and `_list_dlt_table_files` aren't affected by it.

```
# Filters only if pipeline_name provided
if pipeline_name is None or fileparts[0] == pipeline_name:
    yield filepath, filepart
```